### DISCUSSION TITLE
Using Oracle Vector store

### DISCUSSION THREAD

**Discussion by ahhany** [arc53/DocsGPT#980] [Ideas]
Can we use Oracle DB 23 ai as a vector store?

**Comment by dartpain** [comment]
Currently its not supported out of the box, but can be easily added by creating a file here
https://github.com/arc53/DocsGPT/tree/main/application/vectorstore

Will require concrete class similar to others.

Also looks like langchain supports it, which will make integration extra easy.
https://python.langchain.com/v0.2/docs/integrations/vectorstores/oracle/#load-documents
